js/index.js

```diff
@@ -128,19 +137,41 @@ $(function() {
         var iframe = $iframe.get(0).contentWindow.document;
 
         iframe.open();
-        iframe.write(
-            [ '<!DOCTYPE html>'
-            , '<html>'
-            , '  <head>'
-            , '    <title>Try PureScript!</title>'
-            , '    <link rel="stylesheet" href="css/style.css">'
-            , '  </head>'
-            , '  <body>'
-            , '    <div id="console"></div>'
-            , '  </body>'
-            , '</html>'
-            ].join('\n')
-        );
+        if ($('input[name=backend_inputs]').filter(':checked').val() === "thermite") {
+            iframe.write(
+              [ '<!DOCTYPE html>'
+              , '<html>'
+              , '  <head>'
+              , '    <title>Try Thermite</title>'
+              , '    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">'
+              , '    <link rel="stylesheet" href="css/style.css">'
+              , '  </head>'
+              , '  <body>'
+              , '    <div id="console"></div>'
```

Comment:
I'm not sure if we really need this virtual console for Try Thermite. I'd be fine with going back to using the regular console actually.